tests/testthat/test-Median.R

```diff
@@ -0,0 +1,12 @@
+context("Median")
+
+test_that("Test fail", {
+  expect_error(Median(iris))
+})
+
+test_that("Test output", {
+  expect_equal(Median(c(FALSE, TRUE, TRUE, NA)), TRUE)
+  expect_equal(Median(c("string")), "string")
```

Comment:
Why is that possible? Should't the Median() only take numeric data as input?

Reply:
So I went with the original functionality of someR::Median(), which itself reflects the functionality of base::median(). But I also wonder if we for us it would make more sense to restrict the functionality of CTUHelpeR::Median() to only numeric values. @benkertp what do you think?

Reply:
My experience with this function is limited so I think @benkertp 's opinion is the important one.

Reply:
I have rethought this and think that since we simply want the default behavior with `na.rm = TRUE` we can just stick with how it is.

Reply:
The wrapper functions Sum, Median, Average, ... are only needed for the tables packages which need the "na.rm=TRUE" to be called as an argument. we should only used these functions in that context. So I suggest to restring these to restrict the use to numeric vectors only using stopifnot() or similar in the faction body and be adapting the tests accordingly. Agree?

Reply:
ok, @markomi can you add that? Then I think @Inessakraft can merge.

Reply:
Sounds good. Thanks @benkertp